scripts/test_ci.sh

```
@@ -0,0 +1,982 @@
+			log_info "Cleaning up imported secondary wallet: $IMPORTED_WALLET2"
+			$BINARY keys delete $IMPORTED_WALLET2 --keyring-backend test -y 2>/dev/null || true
+		fi
+	fi
+}
+
+# Set trap to cleanup wallet on script exit
+trap cleanup_wallet EXIT
+
+# Source tx flags
+if [ -n "$ZSH_VERSION" ]; then
+	TXFLAG=(--node $RPC --chain-id $CHAIN_ID --gas-prices 1$DENOM --gas auto --gas-adjustment 1.4 -y -b sync --output json)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)